tutorial/tutorial.md

```diff
@@ -581,4 +581,4 @@ If you're still itching for more, why not try something on your own? Here are ju
 	* For an added challenge, if the message is longer than the display, make it scroll!
 * Change the `#define` lines to variables and allow the user to change them on the fly with a `Start` menu
 * Make the device show something special or play a special sound when you enter the Konami code
-* Have the device play .mod sound files instead of .wav and let the user cycle through songs with the press of a button.
\ No newline at end of file
+* Have the device play .mod sound files instead of .wav and let the user cycle through songs with the press of a button.
```

Comment:
not sure what happened here.  github's browser edit didn't like the end of the file I guess?